src/script/sign.cpp

```diff
@@ -294,18 +251,16 @@ namespace
 struct Stacks
```

Comment:
Wonder if we should also change the name of this struct from `Stacks` to `Stack`